docs/source/pages/howToWorkWithPlatform/adminApplication/integration/integrationPage.rst

```diff
@@ -24,31 +27,22 @@ In second field selection customer will provide connection protocol, depends on
 block.
 In the third field customer can provide email address where will be send all notification which related to integrations. Also this email will be use in second default rule.
 
-Connection details(3)
-=====================
-
-Data in connection details (3) block is automatically generated for connection protocols FTPS and SFTP except password, it will be generated after save button click, credential from this block should be used for connect to ftp server. Customer can copy each parameter by clicking on copy button.
-Customer can't change any parameters except password for FTPS and SFTP. To change password customer should click on reset button, after that we will generate new password.
+Connection details (3)
+======================
 
-For AS2 protocol only Self URL and Self certificate will be created, the rest of the fields need to be filled in.
+#. For SFTP protocol all data except in connection details (3) block is automatically generated. Password is generated after Save button clicked. Credential from this block should be used for connection to the FTP server. Each parameter can be copied by clicking on copy button. New password can be created by Reset button.
```

Comment:
Plot of the sentance is different from original one! Please compare original and modified sentences.